app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardScheduleStep/PlanWizardScheduleStep.js

```diff
@@ -4,42 +4,73 @@ import { Field, reduxForm } from 'redux-form';
 import { Form, Icon } from 'patternfly-react';
 import { FormField } from '../../../../../common/forms/FormField';
 
-export const PlanWizardScheduleStep = ({ targetProvider, migration_plan_choice_radio }) => (
-  <Form className="form-horizontal">
+export const PlanWizardScheduleStep = ({ targetProvider, migration_plan_type_radio, migration_plan_choice_radio }) => (
+  <Form className="form-vertical">
     <Field
-      name="migration_plan_choice_radio"
-      label={__('Run migration plan')}
+      name="migration_plan_type_radio"
       component={FormField}
       type="radio"
-      labelWidth="3"
+      label={__('Migration Type')}
+      inline_label={false}
+      controlWidth={12}
       options={[
         {
-          name: __('Save migration plan to run later'),
-          id: 'migration_plan_later'
+          id: 'migration_type_cold',
+          name: __('Cold Migration - Source VMs are shut down while all of the VM data is migrated.')
         },
         {
-          name: __('Start migration immediately'),
-          id: 'migration_plan_now'
+          id: 'migration_type_warm',
+          name: __(
```

Comment:
We need a `// prettier-ignore` here to prevent the `__(...)` from being broken into multiple lines

Reply:
Fixed.